Only in src/ya-market: .DS_Store
diff -r -c target/ya-market/src/api/provider-api.ts src/ya-market/src/api/provider-api.ts
*** target/ya-market/src/api/provider-api.ts	2020-09-07 10:44:28.000000000 +0200
--- src/ya-market/src/api/provider-api.ts	2020-09-07 10:45:04.000000000 +0200
***************
*** 22,28 ****
  // @ts-ignore
  import { Agreement } from '../../src/models';
  // @ts-ignore
! import { AgreementEvent | ProposalEvent | PropertyQueryEvent } from '../../src/models';
  // @ts-ignore
  import { ErrorMessage } from '../../src/models';
  // @ts-ignore
--- 22,28 ----
  // @ts-ignore
  import { Agreement } from '../../src/models';
  // @ts-ignore
! import { AgreementEvent, ProposalEvent, PropertyQueryEvent } from '../../src/models';
  // @ts-ignore
  import { ErrorMessage } from '../../src/models';
  // @ts-ignore
diff -r -c target/ya-market/src/api/requestor-api.ts src/ya-market/src/api/requestor-api.ts
*** target/ya-market/src/api/requestor-api.ts	2020-09-07 10:44:28.000000000 +0200
--- src/ya-market/src/api/requestor-api.ts	2020-09-07 10:44:57.000000000 +0200
***************
*** 30,36 ****
  // @ts-ignore
  import { Proposal } from '../../src/models';
  // @ts-ignore
! import { ProposalEvent | PropertyQueryEvent } from '../../src/models';
  /**
   * RequestorApi - axios parameter creator
   * @export
--- 30,36 ----
  // @ts-ignore
  import { Proposal } from '../../src/models';
  // @ts-ignore
! import { ProposalEvent, PropertyQueryEvent } from '../../src/models';
  /**
   * RequestorApi - axios parameter creator
   * @export
