Only in src/ya-activity: .DS_Store
diff -r -c target/ya-activity/src/api/provider-api.ts src/ya-activity/src/api/provider-api.ts
*** target/ya-activity/src/api/provider-api.ts	2020-09-07 10:44:23.000000000 +0200
--- src/ya-activity/src/api/provider-api.ts	2020-09-07 10:44:46.000000000 +0200
***************
*** 22,28 ****
  // @ts-ignore
  import { ActivityState } from '../../src/models';
  // @ts-ignore
! import { CreateActivity | DestroyActivity | GetActivityState | GetActivityUsage } from '../../src/models';
  // @ts-ignore
  import { ErrorMessage } from '../../src/models';
  /**
--- 22,28 ----
  // @ts-ignore
  import { ActivityState } from '../../src/models';
  // @ts-ignore
! import { CreateActivity, DestroyActivity, GetActivityState, GetActivityUsage } from '../../src/models';
  // @ts-ignore
  import { ErrorMessage } from '../../src/models';
  /**
